survey_one_choice_per_column/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Survey One Choice Per Column
+============================
+
+This module adds an option 'One Choice Per Column' to the matrix type of the matrix question type.
+
+Usage
+=====
+
+* go to Marketing -> Surveys -> Surveys;
```

Comment:
@tarteo Thanks for the contribution
Please use enumerated list ('#.') to describe procedure